bok_choy/browser.py

```diff
@@ -230,6 +231,12 @@ def _local_browser_class(browser_name):
             browser_kwargs = {
                 'firefox_profile': firefox_profile,
             }
+            if os.environ.get('SELENIUM_FIREFOX_PATH', None):
+                binary_kwarg = {
```

Comment:
Is there a benefit to creating this as a separate dict and then updating the other one over just assigning the value directly?

(e.g. `browser_kwargs['firefox_binary'] = FirefoxBinary(firefox_path=os.environ.get('SELENIUM_FIREFOX_PATH'))`)